main/feed_data.R

```diff
@@ -0,0 +1,10 @@
+library(plyr)
+library(readr)
+
+# the combined training data nad test data used in the NIER2019 paper
```

Comment:
nit: nad -> and
You could also have a link or citation to the paper here

Reply:
haha fixed.